config/initializers/monkey_patching.rb

```diff
@@ -0,0 +1 @@
+require "time_extensions"
```

Comment:
Let me know what you think about this: I've created an initializer to monkey patch some methods onto the `Time`/`DateTime`/`TimeWithZone` classes.